Source/Fuse.Navigation/Tests/UX/RouteParamValidation.ux

```diff
@@ -32,6 +32,15 @@
 			router.goto("page5", big_object);
 		};
 
+		module.exports.goto_test6 = function () {
+			var x = {
+				hello:"w3rld",
+				number:1338,
```

Comment:
So 1337 with objects? 1337++

Reply:
I just wanted to modify the values from the previous test. 1337 is just as arbitrary.